src/traces/funnel/attributes.js

```diff
@@ -23,6 +24,9 @@ module.exports = {
 
     hovertext: barAttrs.hovertext,
     hovertemplate: barAttrs.hovertemplate,
```

Comment:
Let's add a `keys` properties so that `percentInitial` and friends show up in our docs.  Example: 

https://github.com/plotly/plotly.js/blob/3e3b22dc87d9806ec2d1b82e8d29f572263ce155/src/traces/histogram/attributes.js#L211-L213